lib/adal-angular.js

```diff
@@ -243,13 +248,16 @@ if (typeof module !== 'undefined' && module.exports) {
                     acquireToken: function (resource) {
                         // automated token request call
                         var deferred = $q.defer();
-                        _adal._renewActive = true;
+                        var isLoginResource = resource === _adal.config.loginResource;
+                        //Renew active is only used for login scenarios so only set it when fetching the login resource.
+                        _adal._renewActive = isLoginResource ? true : _adal._renewActive;
```

Comment:
I just didnt want to change too much of their code to avoid merging issues later